package.json

```diff
@@ -12,13 +16,23 @@
   "keywords": [
     "ecosystem:cordova",
     "cordova-android",
-    "cordova-ios"
+    "cordova-ios",
+    "cordova",
+    "rollbar",
+    "plugin"
   ],
   "engines": [
     {
       "name": "cordova",
       "version": ">=3.6.0"
     }
   ],
+  "scripts": {
+    "prepare": "cp node_modules/rollbar/dist/rollbar.min.js lib/rollbar.js",
+    "test": "echo 'Error: no test specified'"
```

Comment:
Are there no tests for this SDK yet or are you just not using the `test` command?

Reply:
Yes, as discussed in your meeting last week.